routes/chats/ratingFunction.js

```diff
@@ -0,0 +1,21 @@
+module.exports = makeRating;
+
+const { models } = require('../../models/index');
+
+async function makeRating(rating, socket) {
+  console.log('++++++++++++++++++++HERE=======================');
```

Comment:
Do we need the console.log for anything moving forward or can we remove it?

Reply:
☝️